src/metpy/plots/skewt.py

```diff
@@ -286,6 +289,10 @@ def __init__(self, fig=None, rotation=30, subplot=None, rect=None, aspect=80.5):
             Aspect ratio (i.e. ratio of y-scale to x-scale) to maintain in the plot.
             Defaults to 80.5. Passing the string ``'auto'`` tells matplotlib to handle
             the aspect ratio automatically (this is not recommended for SkewT).
+        show_heights : boolean, optional
```

Comment:
```suggestion
        show_heights : bool optional
```
if we keep this interface, this should fix the doc builds yelling at you!